docs/ena_readme.md

```diff
@@ -0,0 +1,72 @@
+## Run GGP using ENA data
+
+ENA is very useful resource for MAGs/bins generation because it [structured](https://www.ebi.ac.uk/training/online/courses/ena-quick-tour/what-is-ena/how-is-ena-structured/) raw-reads and retrieved assemblies in study format.
```

Comment:
```suggestion
ENA is a very useful resource for MAGs/bins generation because it [structures](https://www.ebi.ac.uk/training/online/courses/ena-quick-tour/what-is-ena/how-is-ena-structured/) raw reads and assemblies by study.
```
not sure if this is what you mean